src/reimer_arenkiel_lab_to_nwb/embargo2024/embargo2024_convert_session.py

```diff
@@ -1,71 +1,87 @@
 """Primary script to run to convert an entire session for of data using the NWBConverter."""
+
 from pathlib import Path
 from typing import Union
-import datetime
 from zoneinfo import ZoneInfo
+from natsort import natsorted
 
 from neuroconv.utils import load_dict_from_file, dict_deep_update
 
 from reimer_arenkiel_lab_to_nwb.embargo2024 import Embargo2024NWBConverter
 
 
-def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str, Path], stub_test: bool = False):
+def session_to_nwb(
+    data_dir_path: Union[str, Path], output_dir_path: Union[str, Path], session_date:str, session_id: str, subject_id:str, stub_test: bool = False
+):
 
     data_dir_path = Path(data_dir_path)
+    folder_path = data_dir_path / Path(session_date)
+
     output_dir_path = Path(output_dir_path)
     if stub_test:
         output_dir_path = output_dir_path / "nwb_stub"
     output_dir_path.mkdir(parents=True, exist_ok=True)
 
-    session_id = "subject_identifier_usually"
-    nwbfile_path = output_dir_path / f"{session_id}.nwb"
+    nwbfile_path = output_dir_path / f"{subject_id}_{session_id}.nwb"
 
     source_data = dict()
     conversion_options = dict()
 
-    # Add Recording
-    source_data.update(dict(Recording=dict()))
-    conversion_options.update(dict(Recording=dict()))
-
-    # Add LFP
-    source_data.update(dict(LFP=dict()))
-    conversion_options.update(dict(LFP=dict()))
-
-    # Add Sorting
-    source_data.update(dict(Sorting=dict()))
-    conversion_options.update(dict(Sorting=dict()))
-
-    # Add Behavior
-    source_data.update(dict(Behavior=dict()))
-    conversion_options.update(dict(Behavior=dict()))
+    list_of_fov_boundaries = [[0,400], [450, 850], [900,1300]]
```

Comment:
I would include some information about the FOV boundaries in the `TwoPhotonSeries` description, also instead of naming it `TwoPhotonSeries...FOV0` can it be something more descriptive like FOVLeft?

Reply:
Okay good, makes sense.